src/main/java/com/atlassian/jira/cloud/jenkins/deploymentinfo/pipeline/JiraSendDeploymentInfoStep.java

```diff
@@ -42,6 +42,7 @@
     private String state;
     private List<String> serviceIds = new ArrayList<>();
     private Boolean enableGating = Boolean.FALSE;
+    private List<String> issueKeyIds = new ArrayList<>();
```

Comment:
Can we call this just `issueKeys` everywhere?

Reply:
I take it you mean everywhere as in all files?